format_spec/filter_pipeline.md

```diff
@@ -80,6 +80,9 @@ For the `TILEDB_FILTER_DELTA` and `TILEDB_FILTER_DOUBLE_DELTA` compression filte
 | Compression level | `int32_t` | Ignored |
 | Reinterpret datatype | `uint8_t` | Type to reinterpret data prior to compression. |
 
+> [!NOTE]
+> Prior to version 20, the _Reinterpret datatype_ field was not present for the double delta filter. Also prior to version 19, the same field was not present for the delta filter.
```

Comment:
The delta filter did not "exist" prior to version 19, but as said above filters are not introduced in specific format versions so we are specific that the filter _option_ was introduced.

Reply:
Would it maybe be better to phrase this like this instead:

> The _Reinterpret datatype_ field was added to the delta filter in version 19 and the double delta filter in version 20.

Reply:
My preference is to document the latest version in the main specification[^1] and mention behavior in prior versions in notes like this.

[^1]: An exception is simple additions and removals where I used the "New/Removed in version **" phrasing. Adding a "New in version 19 for the delta filter and new in version 20 for the double delta filter" would be too much for an inline comment.